src/Microsoft.ML.LightGbm/WrappedLightGbmBooster.cs

```diff
@@ -162,10 +162,10 @@ private static double[] GetDefalutValue(double[] threshold, UInt32[] decisionTyp
             return ret;
         }
 
-        private static bool FindInBitset(UInt32[] bits, int start, int end, int pos)
+        private static bool FindInBitset(UInt32[] bits, int start, int size, int pos)
```

Comment:
[https://github.com/Microsoft/LightGBM/blob/90127b526be8b13bdf840effacde9e61fa765a6f/include/LightGBM/utils/common.h#L827](https://github.com/Microsoft/LightGBM/blob/90127b526be8b13bdf840effacde9e61fa765a6f/include/LightGBM/utils/common.h#L827)
this is original function in LightGBM.